### PR TITLE
Set up GitHub Actions for testing and releasing

### LINKED ISSUE #74: Use GitHub Actions for releasing package distribution
https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

### PR DESCRIPTION
**Pull request by takuti**
- Replace Travis with GH Actions
- Automate PyPI release, which closes #74

### CODE REVIEW

**Comment by takuti** [comment]
@chezou Can you take a look at the updates? Note that AppVeyor didn't run in this repository for a long time...just deleting the config files, and making subtle modifications so that unit tests pass both on Ubuntu and Windows.

File: pytd/writer.py [hunk 1]
```diff
@@ -437,6 +438,8 @@ def write_dataframe(self, dataframe, table, if_exists, fmt="csv", keep_list=Fals
         self._bulk_import(table, fp, if_exists, fmt)
 
         fp.close()
+        if fmt == "csv":
```
Comment (chezou) [inline pytd/writer.py]: To ensure deleting temporary when pytd gets an exception, how about using `contextlib.ExitStack()` or `contextlib.contextmanager`?
https://docs.python.org/3/library/contextlib.html#replacing-any-use-of-try-finally-and-flag-variables
https://docs.python.org/ja/3/library/contextlib.html#contextlib.contextmanager

Comment (takuti) [reply pytd/writer.py]: Nice catch! How about this ec12cb8e9b8874dfdfe2fccaf71f6a580bab8944 with a new test case d02a158783f6258bdc1327db0a7c743e1086ae5a?

Comment (chezou) [reply pytd/writer.py]: Or, we may try to use `BytesIO` instead of writing temporary file.

**Pull request merged**